lib/HeroBonus.h

```diff
@@ -285,7 +285,6 @@ class DLL_LINKAGE CAddInfo : public std::vector<si32>
 	BONUS_NAME(SPELL_DAMAGE) /*val = value*/\
 	BONUS_NAME(SPECIFIC_SPELL_DAMAGE) /*subtype = id of spell, val = value*/\
 	BONUS_NAME(SPECIAL_BLESS_DAMAGE) /*val = spell (bless), additionalInfo = value per level in percent*/\
-	BONUS_NAME(MAXED_SPELL) /*val = id. deprecated in favour of SPECIAL_FIXED_VALUE_ENCHANT*/\

```

Comment:
Interesting if we break save compatibility here as bonus enum numeration may shift?

Reply:
If yes I would rename bonus instead as DEPRECATED_MAXED_SPELL or UNUSED_MAXED_SPELL and remove when we break save compatibility once again. (Or do we plan to do this in 1.2?). Or at least increase serialization version.